bukuserver/server.py

```diff
@@ -33,7 +35,7 @@
 STATISTIC_DATA = None
 
 def _fetch_data():
-    url = request.data.get('url')
+    url = request.form.get('url')
```

Comment:
There's also auto-parsing of `request.data`, but the same exact functionality appears to be implemented in `request.form` anyway (either in Flask or Flask-Admin, I don't remember which)